README.md

```diff
@@ -220,6 +221,49 @@ module.exports = function(environment) {
 
 Note if you want to disable right and left in-viewport triggers, set these values to `Infinity`.
 ```
+
+### Modifiers
+
+Using with [Modifiers](https://blog.emberjs.com/2019/03/06/coming-soon-in-ember-octane-part-4.html) is easy.
+
+1.  Install [@ember/render-modifiers](https://github.com/emberjs/ember-render-modifiers)
+2.  Use the `did-insert` hook inside a component
+3.  Wire up the component like so
+```js
+import Component from '@ember/component';
+import { set } from '@ember/object';
+import InViewportMixin from 'ember-in-viewport';
+
+export default Component.extend(InViewportMixin, {
+  tagName: '',
+
+  // if you do have a tagName ^^, then you can use `didInsertElement` or no-op it.  You choose
+  // didInsertElement() {},
+  didInsertNode(element, [instance]) {
+    instance.watchElement(element);
+  },
+
+  init() {
+    this._super(...arguments);
+
+    set(this, 'viewportSpy', true);
+    set(this, 'viewportTolerance', {
+      bottom: 300
+    });
+  },
+
+  didEnterViewport() {
+    this.infinityLoad();
+  }
+});
+```
+
+```hbs
+<div {{did-insert this.didInsertNode this}}>
```

Comment:
I like that it uses the existing implementation in a clean way, but I wonder if there's a way to write the modifier such that it doesn't need the entire component instance? I had thought that the benefit of modifiers is that they can operate on DOM nodes independently of their associated component classes.

My initial thought at implementing (before i knew about `@ember/render-modifiers`, was to wire up an `IntersectionObserver` with the `element` directly. 

I don't quite get how modifier managers work at the moment, so I wasn't able to understand what the `did-insert` modifier is actually doing under the hood

Reply:
Good point.  Added a note about a future add `did-enter-viewport` modifier + a service (instead of a mixin).  Will think about this over the following months.  Thanks for taking a look!